tests/unit/lambdas/repos.test.js

```diff
@@ -0,0 +1,117 @@
+/* eslint-env jest */
+import ReposLambda from '../../../src/functions/oss/repos'
+
+import ossUtils from '../../../src/functions/oss/utils'
+ossUtils.updateEntry = jest.fn().mockImplementation(() => Promise.resolve(null))
+
+process.env.LAMBDA_ENV = 'production'
+
+const sameRepoOne = {
+  url: 'https://github.com/yldio/fake-repo',
+  nameWithOwner: 'yldio/fake-repo',
+  descriptionHTML: '<div>This is a fake repo.</div>',
+  starCount: 10924,
+  pullRequestCount: 4
+}
+
+const differentRepoOne = Object.assign({}, sameRepoOne)
+differentRepoOne.pullRequestCount = 6
```

Comment:
This is the same thing but keeps it a bit neater.
```
const differentRepoOne = {
    ...sameRepoOne,
    pullRequestCount: 6
}
```

Reply:
Yes it is. Good call. Thanks 👍